### ISSUE TITLE
Question about PTO-RPM

### ISSUE THREAD

**Issue by mrPassion78** [modelleicher/FS19_realManualTransmission#83]
Hi there, i was wondering if it would be possible   to add the function of the PTO-rpm, so it revs up the engine rpm to te set PTO-rpm, like in the MR mod in fs17.

I have no clue about the difficulty of this withing the LUA, and how much work that would be IF...it would be possible ..

What i want to achieve with it is: 
If the tractor has the forage mixer behind it, and i turn it on, that the rpm would go up ( aslong as the clutch is engaged or in neutral ), to a set value in an new attribute .

in real life , every implement would work at a different pto-rpm, but i would love it when in would auto rev up at all, and you can adjust it to different setting with the hand trottle.

I hope i`m not asking for alot of work here,

thnx in advance for reading this.
Greetings pascal

**Comment by modelleicher** [comment]
Hi,

If I understand correctly you want a feature that automatically revs up the engine if a PTO implement is engaged. Thats a basegame feature and a unrealistic one too, but needed if there is no hand throttle.
But since RMT has hand throttle I purposely removed that function since no tractor IRL automatically increases engine RPM if the pto is engaged I think.. unless for some really modern ones with pre-set rpm settings but those don't automatically increase when turning on the PTO either they usually have special function key's like Makros to do a bunch of stuff automatically when the implement is turned on.

So in short.. unless there is a realism reason for this I don't intend to add it back in.

**Comment by mrPassion78** [comment]
You are 100% right, and it does not work that way in real life..indeed, 
I didn`t think that far trough, that it might not even exist IRL..
I`ll close this toppic.. 

But please keep up the awesome work you do on this mod.. thnx a millon for it
